backend/src/migrations/20240221213010_add_user_achievements_table.ts

```diff
@@ -0,0 +1,35 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'user_achievements';
+
+const ColumnName = {
+    USER_ID: 'user_id',
+    ACHIEVEMENT_ID: 'achievement_id',
+    DATE_ACHIEVED: 'date_achieved',
+};
+
+async function up(knex: Knex): Promise<void> {
+    return knex.schema.createTable(TABLE_NAME, (table) => {
+        table.increments().primary();
```

Comment:
So, are we not including a primary key in this table, or does this implicitly create a primary key somehow?